doc/sphinx/language/extensions/match.rst

```diff
@@ -316,21 +317,13 @@ a warning is issued in such situations.
 
    .. example::
 
-      .. coqtop:: none
-
-         Set Warnings "-unused-pattern-matching-variable".
-
-      .. coqtop:: all
+      .. coqtop:: all warn
 
          Definition is_zero (o : option nat) := match o with
-         | Some 0 => true
+         | Some _ => true
```

Comment:
This is a dumb example. Something like:
```coq
Definition is_some (o : option nat) :=
  match o with
  | Some x => true
  | None => false
  end.
```
is better

Reply:
This one doesn't trigger the warning though.

Reply:
Ah ok then I misread.